.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
This somehow leaked in. Shouldn't be here.

Reply:
I don't see it today. How do I remove it?

Reply:
I did for you before since removing a settings file could be tricky.